### PR TITLE
test: Adapt TestMachines to fixed bug in fedora-33

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
This is expected to fail on fedora-33 and succeed on fedora-33@bots#1240, which has the new fedora-33 image.

File: test/verify/check-machines [hunk 1]
```diff
@@ -1566,10 +1566,13 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
 
         # We don't want to use start_vm == False because if we get a separate install phase
         # virt-install will overwrite our changes.
-        # https://bugzilla.redhat.com/show_bug.cgi?id=1818089
-        # After shutting it off virt-install will restart the domain and exit because of the above bug
-        # Wait for virt-install to define the VM and then stop it in real
-        self.machine.execute("virsh destroy pxe-guest")
+
+        if self.machine.image not in [ "fedora-33" ]:
```
Comment (martinpitt) [inline test/verify/check-machines:1570]: Eventually we want to invert the condition (as the fix trickles into more OSes), and perhaps factor it out. But we will notice when the fix trickles into other OSes, so we won't forget about this.

**Review by martinpitt — APPROVED** [review]
Thanks! Tests have the last word, but this LGTM.

**Comment by mvollmer** [comment]
TestMachines.testConfigureBeforeInstall has managed to pass on the current fedora-33 after failing twice...

**Comment by mvollmer** [comment]
fedora-32/firefox seems to have infrastructure trouble in general, but it's not relevant here.

**Pull request merged**